scipy/spatial/transform/rotation.py

```diff
@@ -657,3 +657,105 @@ def __mul__(self, other):
         if self._single and other._single:
             result = result[0]
         return self.__class__(result, normalized=True)
+
+    def __getitem__(self, val):
```

Comment:
Definitely needs to be documented. You should explain that only one index should be given and what it does.

Reply:
But we're not using a single index. In my code, I'm using is for slicing and reordering.

Reply:
Ah, ok.